functions.php

```diff
@@ -989,7 +989,12 @@ function get_jft($sms = false) {
     if ($sms == true) {
         $without_htmlentities = html_entity_decode($trim_results);
         $without_extranewlines = preg_replace("/[$preg_search_lang]+/", "$preg_replace_lang", $without_htmlentities);
-        return $without_extranewlines;
+        $message = sms_chunk_split($without_extranewlines);
+        $finalMessage  = array();
+        for ($i = 0; $i < count($message); $i++) {
+            $finalMessage[] = "(" .($i + 1). " of " .count($message). ")\n" .$message[$i];
```

Comment:
i'm not sure this will work?  did you test this out before you made this PR?

Reply:
I did test
![image-1](https://user-images.githubusercontent.com/34245618/47259580-06804100-d479-11e8-8ba0-59292bdbe07f.jpg)

Reply:
ok for code style consistency please use `array_push` versus `$finalMessage[] = `